offchain-computing/gramine/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM docker-regis.iex.ec/iexec-graphene-base:0.6.0
+
+RUN apt-get update && apt install -y python3 apktool gcc musl-dev
```

Comment:
Could we purge APT cache here ?
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments